README.md

```diff
@@ -5,7 +5,7 @@
 
 ## Built With
 
-- Html, js.
+- Html, Js, Css.
 - Linters.
 
 ## Authors
```

Comment:
- [ ] Please add your collaborator @shuhad786 name and social links to the Authors section to make your readme file more professional.